src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,34 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+from z3c.form import interfaces
+ 
+from zope import schema
+from zope.interface import Interface
+ 
+from zope.i18nmessageid import MessageFactory
+ 
+_ = MessageFactory('experimental.safe_html_transform')
+ 
+ 
+class IExperimenatalSafe_Html_TransformSettings(Interface):
+"""Global safe_html_transform settings. This describes records stored in the
+	configuration registry and obtainable via plone.registry.
+"""
+ 
+    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
```

Comment:
It doesn't matter in Plone if those files are in a different folder. You can keep your current structure. You just have to replace the Class. Though, you need to understand what you are doing. If you are just copy/pasting code it won't work. See https://pypi.python.org/pypi/plone.app.registry for details about the registry control panels.